tests/framework/extensions/provisioning/verify.go

```diff
@@ -158,6 +160,11 @@ func VerifyCluster(t *testing.T, client *rancher.Client, cluster *steveV1.SteveA
 	podResults, podErrors := pods.StatusPods(client, status.ClusterName)
 	assert.Empty(t, podErrors)
 	assert.NotEmpty(t, podResults)
+
+	if clustersConfig.SSHTests != nil {
+		tests := clustersConfig.SSHTests
```

Comment:
nit; you could just directly enter this when calling the verifySSHTests function instead of declaring it.